### PR TITLE
Video subsection creation

### PR DESCRIPTION
**Pull request by SgtCoin**
Initial pull request for YouTube video transcriptions.

### CODE REVIEW

**Comment by SgtCoin** [comment]
fleek-ci failed with `2022-09-30T15:51:27.388Z: Error: Error building site: "/workspace/layouts/_default/_markup/render-image.html:8:20": execute of template failed: template: _default/_markup/render-image.html:8:20: executing "_default/_markup/render-image.html" at <$image.Resize>: error calling Resize: error calling resources.GetRemote: Get "https://i.imgur.com/dN0x2Ye.jpg": x509: certificate signed by unknown authority`

File: content/en/storage-provider/videos/actors.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+- [What are Filecoin actors](#what-are-filecoin-actors)
+- [The Actor Model and Blockchain Consensus](#the-actor-model-and-blockchain-consensus)
+- [Actor Code](#actor-code)
+- [Filecoin is complicated](#filecoin-is-complicated)
+- [Future Actors and Two Layers](#future-actors-and-two-layers)
+- [The FVM’s Impact](#the-fvms-impact)
+- [Verifiable Storage](#verifiable-storage)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
```
Comment (johnnymatthews) [inline content/en/storage-provider/videos/actors.md:26]: The TOC gets auto generated on each page already. Don't think this is necessary:

image

File: content/en/storage-provider/videos/actors.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+### What are Filecoin actors
+
+Filecoin actors are hard-coded programs that run on the Filecoin blockchain. If you understand Ethereum style smart contracts, you more or less understand Filecoin actors. Actors are written ahead of time (as of the time of writing) by Protocol Lab engineers. Actors are a key aspect of how the Filecoin blockchain protocol operates. 
+
+### The Actor Model and Blockchain Consensus
+
+Blockchain consensus is a way to figure out a shared state. Filecoin uses incremental block appends to accomplish this. You will see a bunch of blocks with some forks. These forks are not used for consensus.
+
+![actors1](https://imgur.com/6p3WEbb.png)
```
Comment (johnnymatthews) [inline content/en/storage-provider/videos/actors.md:36]: If you're adding images, there's a couple of things you need to do:

1. Create a folder within `storage-provider/videos/` called `actors`.
2. Move this `.md` into the `actors` folder and rename it `index.md`
3. Move any images into the `actors` folder.
4. Reference the images from within the `index.md` file like `Alt text`.

**Review by johnnymatthews — CHANGES_REQUESTED** [review]
I've left a couple of comments that need addressing. Essentially, to merge this PR we need to:

1. Remove the TOCs. They get auto generated by Hugo.
2. Use [Page Bundles](https://gohugo.io/content-management/page-bundles/) when including images.

File: content/en/storage-provider/videos/actors.md (hunk 1 above)
Comment (SgtCoin) [reply content/en/storage-provider/videos/actors.md]: Removed the TOC.

File: content/en/storage-provider/videos/actors.md (hunk 2 above)
Comment (SgtCoin) [reply content/en/storage-provider/videos/actors.md]: Thank you for the help with this. I completed this for each video.

**Comment by smagdali** [comment]
These are great. 

Some comments: some general, some specific:

- http://localhost:1313/storage-provider/videos/cryptoecon/ - this one doesn't intro the speaker (with job title) in the opening paragraph. - can you check them all? 
- I'd change the title to be the same as the title of the talk, so this one should be
'Intro to Filecoin Tokenomics', etc.
- I'd say something like "Crypto-economics, often shortened to 'cryptoecon'" in the first paragraph.

- http://localhost:1313/storage-provider/videos/opendata/ - Can include the question at the end in the summary?

On all: 
- can we order them in sequence from introductory to advanced, as per the lifecycle we've been discussing, rather than alphabetical, even if we can't put sections in?

-now that I've seen them on the page together, I think it would help to make it clearer that the written content is a summary of the video. Perhaps something like: "In this video, XX explains:" just below the video of each, before the main text starts.

**Pull request closed without merging**